### PR TITLE
chore: prep the repo for automated releasing

### PR DESCRIPTION
**Pull request by eshanholtz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,47 +14,28 @@ matrix:
       sudo: true
 env:
   global:
-    - CC_TEST_REPORTER_ID=$TRAVIS_CODE_CLIMATE_TOKEN
+    - CC_TEST_REPORTER_ID=$TRAVIS_CODE_CLIMATE_TOKEN SENDGRID_API_KEY=SGAPIKEY
```
Comment (childish-sambino) [inline .travis.yml]: _$SGAPIKEY_?

File: sendgrid/version.py [hunk 2]
```diff
@@ -1,2 +1 @@
-version_info = (6, 1, 0)
```
Comment (childish-sambino) [inline sendgrid/version.py]: Not sure if clients used `version_info` at all.

Comment (thinkingserious) [reply sendgrid/version.py]: It was used to build `__version__` (don't know why though), which is used here: https://github.com/sendgrid/sendgrid-python/blob/master/setup.py#L21

Comment (thinkingserious) [reply sendgrid/version.py]: Ah, need to update this: https://github.com/sendgrid/sendgrid-python/blob/ea796fb14e6c397e156ba37f30d4fdb06675a61f/sendgrid/helpers/mail/email.py#L7

Comment (eshanholtz) [reply sendgrid/version.py]: @thinkingserious `sys.version_info` returns the separated _python_ version information. It seemed like it was only used to build the `__version__` number in `version.py`, per your previous comment.

File: .travis.yml (hunk 1 above)
Comment (eshanholtz) [reply .travis.yml]: hm I was just trying to set a dummy value. Since it's not an actual env var I'm don't think the $ is needed?

Comment (childish-sambino) [reply .travis.yml]: Ah, nevermind,

**Comment by thinkingserious** [comment]
Hello @eshanholtz,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**